### PR TITLE
Contact Section

### PR DESCRIPTION
**Pull request by coker2019**
Added  the contact section to my mobile first portfolio project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -295,6 +295,35 @@ <h3 class="lang">Skills</h3>
...
+        <label for="full-name">
+          <input class="input_name" type="text" name="name" id="full-name" placeholder="Maurice"
+            accept-charset="utf-30">
+        </label>
+        <label>
+          <input class="input_mail" type="email" name="email" placeholder="Olabisi2k11@gmail.com">
+        </label>
+        <label>
+          <textarea class="message_box" name="message" placeholder="write your message here"></textarea>
+        </label>
```
Comment (Lizdev-05) [inline index.html]: Great job so far 👍 . Kindly make sure all the input field has the `required` attribute to activate the validation on submit. 😊

File: index.html [hunk 2]
```diff
@@ -295,6 +295,35 @@ <h3 class="lang">Skills</h3>
...
+
+        <label for="full-name">
+          <input class="input_name" type="text" name="name" id="full-name" placeholder="Maurice"
+            accept-charset="utf-30">
+        </label>
+        <label>
+          <input class="input_mail" type="email" name="email" placeholder="Olabisi2k11@gmail.com">
+        </label>
+        <label>
+          <textarea class="message_box" name="message" placeholder="write your message here"></textarea>
```
Comment (Lizdev-05) [inline index.html]: According to the requirement, the message textarea should have a `max length of "500"` attribute. Kindly add it to it. 😊

File: index.html [hunk 3]
```diff
@@ -295,6 +295,35 @@ <h3 class="lang">Skills</h3>
...
+      or a project that
+      needs coding. I’d love to help with it </p>
+
+    <form action="https://formspree.io/f/xleknroo" method="POST">
+      <form id="fs-frm" class="contact_form" name="simple-contact-form" action="https://formspree.io/f/{form_id}"
+        method="post">
+
+        <label for="full-name">
+          <input class="input_name" type="text" name="name" id="full-name" placeholder="Maurice"
+            accept-charset="utf-30">
```
Comment (Lizdev-05) [inline index.html]: Great job so far, According to the requirement, the message input field for the name should have a max length of "300" attribute. Kindly implement it. 😊

File: index.html [hunk 4]
```diff
@@ -295,6 +295,35 @@ <h3 class="lang">Skills</h3>
...
+        <label>
+          <input class="input_mail" type="email" name="email" placeholder="Olabisi2k11@gmail.com">
+        </label>
+        <label>
+          <textarea class="message_box" name="message" placeholder="write your message here"></textarea>
+        </label>
+        <button class="touch_btn" type="submit">
+         Get in touch
+        </button>
+      </form>
```
Comment (Lizdev-05) [inline index.html]: Kindly make sure the button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. 👇 

image

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @coker2019, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct GitHub Flow 👍

- Good commit history 💰

- Good PR title💯

- All linters passed 🏅

- Personalised README file 👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — APPROVED** [review]
Hi @coker2019 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6a4a5143327075565a70544d4f34765573302f67697068792e676966


## Highlights
✔Good use of github workflow
✔Good PR title
✔No linter errors
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — COMMENTED** [review]
Hi @coker2019 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Portfolio: add a contact form and finish the mobile version.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**